src/components/Courses/Courses.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import SearchBar from './components/SearchBar/Seacrbar';
+import Button from '../../common/Button/Button';
+import CourseCard from './components/CourseCard/CourseCard';
+import CreateCourse from '../CreateCourse/CreateCourse';
+
+import {
+	mockedCoursesList,
+	BUTTON_TEXT,
+	mockedAuthorsList,
+} from '../../constans';
+import filterCourses from '../../helpers/filterCourses';
+import { Course, Author } from '../../interfaces/index';
+
+import './courses.css';
+
+function Courses() {
+	const [renderedComponent, setRenderedComponent] = useState(
+		'courseCardComponent'
+	);
```

Comment:
You can use boolean here, string is a little redundant.
Just display your components like this: 
```
	const [showCreateCourse, setShowCreateCourse] = useState(false);
	...
	return (
	  ...
	  {(showCreateCourse) ? <CreateCourse/> : <CourseList/>}
	  ...
	)
```

Reply:
done